docs/cover.md

```diff
@@ -26,6 +26,7 @@ Shutters are simple representations of blind/roller cover actuators. With XKNX y
 - `travel_time_up` seconds to reach upper end position. Default: 22
 - `invert_position` invert position (payload for eg. set_up() and relative position). Default: False
 - `invert_angle` invert angle. Default: False
+- `device_class` may be used to store the type of cover, e.g. "shutter" for Home-Assistant.
```

Comment:
Would you mind adding a link to the supported device classes?

Reply:
Link had been added